test/tasks/test_gapic.py

```diff
@@ -107,7 +107,7 @@ class GapicConfigMoveTaskTests(unittest.TestCase):
     @mock.patch.object(gapic_tasks.GapicConfigMoveTask, 'exec_command')
     def test_execute(self, exec_command):
         task = gapic_tasks.GapicConfigMoveTask()
-        task.execute('/path/src', ['/path/dest'])
+        task.execute('/path/src', '/path/dest')
```

Comment:
Did the signature of `GapicConfigMoveTask()` change, or was this just wrong before?